src/items.cpp

```diff
@@ -1368,6 +1370,17 @@ void Items::parseItemNode(const pugi::xml_node& itemNode, uint16_t id)
 					break;
 				}
 
+				case ITEM_PARSE_WORTH: {
```

Comment:
Should a check be added to ensure worth is a power of 10? The change gold script won't work with any other values.

Reply:
That does not concern us, whoever wants to implement an strange currency should know that he must create his own script to handle said currencies

Reply:
It should concern us... Over the last few months many diffs were accepted to help those who want to run a server from making mistakes. This is another area we should protect server admins from making a mistake and dealing with the headache of figuring out what is wrong since the system only works with power's of 10. At a minimum that information could be added to the PR description. I myself thought I could use any value until reading the code.